### PR TITLE
OPSEXP-1833 Bump imagemagick package version

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-1833

### CODE REVIEW

**Review by alxgomz — DISMISSED** [review]
That's not very sustainable.
make the suffix part of the version variable, or - if not possible - create a new suffix variable to append in var files.
That way we only have one place where we need to fix things in case we update that release build number

**Comment by slohe1** [comment]
> That's not very sustainable. make the suffix part of the version variable, or - if not possible - create a new suffix variable to append in var files. That way we only have one place where we need to fix things in case we update that release build number

Bumped a new variable "imagemagick_release" under dependencies_version so that its one place for both imagemagick version & release number

File: roles/transformers/vars/CentOS7.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-imagemagick_version: "{{ dependencies_version.imagemagick }}-ci"
+imagemagick_version: "{{ dependencies_version.imagemagick }}-ci-{{ dependencies_version.imagemagick_release }}"
```
Comment (gionn) [inline roles/transformers/vars/CentOS7.yml:2]: I still think that just using`dependencies_version.imagemagick` would be simpler and gives more flexibility on the package version

**Review by gionn — CHANGES_REQUESTED** [review]
looking at the test failures it seems we have a problem with RHEL 8, `test_get_rendition` is always failing on integration tests

then it seems there is a problem running `libreoffice` in the transformers role tests [link](https://github.com/Alfresco/alfresco-ansible-deployment/actions/runs/3487299672/jobs/5834772497#step:7:1667)

**Review by gionn — APPROVED** [review]

**Pull request merged**